src/geolocation/controllers/geocoderController.js

```diff
@@ -0,0 +1,23 @@
+import { geolocationService as geocoder } from '../services';
+
+let instance = null;
+class GeocoderController {
+  constructor(service) {
+    if (!instance) instance = this;
+    this.service = service;
+  }
+
+  addressToCoordinates = async function getCoordinates(request, response) {
```

Comment:
async addressToCoordinates(request, response)

Reply:
If you're losing context you can bind the function to itself in controller consturctor